src/components/pages/user/components/subscription-details.tsx

```diff
@@ -61,8 +61,18 @@ const SubscriptionDetails: React.FunctionComponent<SubscriptionDetailsProps> =
                   </strong>
                   ). You can renew at any time.
                 </p>
-                <p></p>
-                <p></p>
+                <p>
+                  You can also get help by emailing{' '}
+                  <strong>
+                    <a
+                      href="mailto:support@egghead.io"
```

Comment:
mailto links can have a subsject AND body added which should almost always be the case for this type of usage https://mailtolinkgenerator.com/

Reply:
something like:
```
<a
  href={`mailto:support@egghead.io?subject=${encodeURIComponent(
    `Subscription ${subscriptionData.subscription.id} is cancelled`,
  )}&body=${encodeURIComponent(
    `Subscription ${subscriptionData.subscription.id} is cancelled`,
  )}`}
  style={{textDecoration: 'underline'}}
>
  support@egghead.io
</a>
```